### PR TITLE
Add Azure SSA rake tasks

### PR DESCRIPTION
**Pull request by djberg96**
This adds the SSA related tasks for Azure back into the available rake tasks.

### CODE REVIEW

**Comment by Fryguy** [comment]
Not sure why this is in gems pending...I would expect this in the azure provider plugin.

**Comment by djberg96** [comment]
@Fryguy These tasks are for the SSA related stuff, regenerating cassettes and so on.

**Comment by Fryguy** [comment]
There is a way to include rake tasks within the azure plugin.  If tasks are placed within lib/tasks of the plugin gem, then they will be available from the plugin and from manageiq.  If they are put in lib/tasks_private, they will only be available within the gem.  IMO, these tasks should be moved from here into the gem.

@durandom @bdunne Thoughts?

File: Rakefile [hunk 1]
```diff
@@ -3,6 +3,8 @@ require 'rspec/core/rake_task'
 require 'rake'
 require 'rake/testtask'
 
+Rake.add_rakelib 'lib/gems/pending/lib/tasks'
```
Comment (bdunne) [inline Rakefile]: Why is this file in `lib/gems/pending/lib/tasks` rather than `lib/tasks`?

**Comment by bdunne** [comment]
@Fryguy I guess I don't know enough about it to decide where it should live, but I don't like where it currently is. :)

**Comment by durandom** [comment]
Shouldnt those specs and cassettes move to the azure provider?

**Comment by djberg96** [comment]
@durandom Eventually the SSA stuff is going to get split into its own gem. I think that's the plan anyway.

**Comment by chessbyte** [comment]
@djberg96 Eventually is almost here: https://github.com/ManageIQ/manageiq-gems-pending/pull/164

**Comment by djberg96** [comment]
@chessbyte Alright, I'll move them over when it's ready.

**Comment by Fryguy** [comment]
@miq-bot move_issue ManageIQ/manageiq-smartstate

**Comment by Fryguy** [comment]
Derp, of course not :)

**Comment by Fryguy** [comment]
@djberg96 Please recreate this PR in the new repo https://github.com/ManageIQ/manageiq-smartstate

**Pull request closed without merging**